playground/backend/cmd/server/server.go

```diff
@@ -109,6 +118,18 @@ func setupCache(ctx context.Context, appEnv environment.ApplicationEnvs) (cache.
 	}
 }
 
+// setupExamplesCatalog saves precompiled objects catalog from storage to cache
+func setupExamplesCatalog(ctx context.Context, cacheService cache.Cache) error {
```

Comment:
No tests for this method added

Reply:
This goes from [[BEAM-13632]](https://github.com/apache/beam/pull/16493)